independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/classloading/DriverRemover.java

```diff
@@ -0,0 +1,25 @@
+package io.quarkus.bootstrap.classloading;
+
+import java.sql.Driver;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Enumeration;
+import org.jboss.logging.Logger;
+
+public class DriverRemover implements Runnable {
```

Comment:
There are two of these classes. Is it supposed to be one for the runtime and one for the deployment CL?

Reply:
oops, must have copied instead of moved